src/context.ts

```diff
@@ -440,8 +502,8 @@ export default class Context {
 		return assignment;
 	}
 
-	_queueExposure(experimentName, assignment) {
-		const exposureEvent = {
+	private _queueExposure(experimentName: string, assignment: ExperimentData) {
```

Comment:
Type of assignment here should be the new type `Assignment`

Reply:
Done :)